python/cudf/cudf/_lib/pylibcudf/groupby.pyx

```diff
@@ -90,14 +90,39 @@ cdef class GroupBy:
         Whether or not to include null rows in ``keys``. Default is null_policy.EXCLUDE.
     keys_are_sorted : sorted, optional
         Whether the keys are already sorted. Default is sorted.NO.
+    column_order : list[order]
+        Order of each key column if the keys are sorted. Default if not
+        provided uses order.ASCENDING. Ignored if keys_are_sorted is sorted.NO.
```

Comment:
I know this follows the established pattern but why don't we backtick our enum values?

```suggestion
        provided uses ``order.ASCENDING``. Ignored if keys_are_sorted is ``sorted.NO``.
```

Reply:
Done. Mostly we don't do it so that the docstrings are readable in the REPL more easily.